scripts/api/post-install-api-block.sh

```diff
@@ -1,11 +1,27 @@
 #!/bin/bash
 
-# Run this script post-installation, to block all the settings that 
-# should not be available to the general public in a production Dataverse installation.
+# This script can be run on a system that was set up with unrestricted access to 
+# the sensitive API endpoints, in order to block it for the general public.
+
+# First, revoke the authentication token from the built-in user: 
+curl -X DELETE $SERVER/admin/settings/BuiltinUsers.KEY
```

Comment:
@landreev ah, so this is why you aren't bothering to block the "builtin-users" endpoint in setup-all.sh. You're deleting the `BuiltinUsers.KEY` setting. I had previously added warnings about the "builtin-users" endpoint at http://guides.dataverse.org/en/4.2.4/installation/config.html#blocking-api-endpoints but I think what I wrote there is fine and still applies to 4.3. No changes needed.